mms/model_loader.py

```diff
@@ -112,7 +112,14 @@ def load(self, model_name, model_dir, handler, gpu_id, batch_size):
         if module_name.endswith(".py"):
             module_name = module_name[:-3]
 
+        # Remove mms dir from python path to avoid module name conflict.
+        mms_path = os.path.dirname(os.path.realpath(__file__))
+        while mms_path in sys.path:
+            sys.path.remove(mms_path)
+
         module = importlib.import_module(module_name)
+        sys.path.append(mms_path)
```

Comment:
How does this handle runtime "import"? 
What happens if we import a module at inference-runtime.
```
def inference(ctx, data):
    import service as svc
    svc.process_inference(data)
```

Reply:
You are right, we should not add it back.
All mms code import with "mms." explicitly, we don't need mms path in the python path.
I will move the code to beginning of model_server_worker.py.